lib/Alchemy/Phrasea/Controller/Thesaurus/ThesaurusController.php

```diff
@@ -2128,7 +2128,7 @@ private function getXMLTerm($bid, $id, $typ, $piv, $sortsy, $sel, $nots, $acf)
                                 $sy = $sy_list->appendChild($ret->createElement("sy"));
 
                                 $sy->setAttribute("id", $n->getAttribute("id"));
-                                $sy->setAttribute("v", $t = $n->getAttribute("v"));
+                                $sy->setAttribute("v", htmlspecialchars($t = $n->getAttribute("v")));
```

Comment:
Same as above : we should not pre-escape for xml (setAttribute does it). We should only escape when we generate html.